README.md

```diff
@@ -27,3 +26,12 @@ about it at the start of each recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+
+## Answer
+Invariant: Begin with 0 and 1 as the initial values in the Fibonacci number sequence. Then, repeatedly sum the two most recent counts and append the result to an array to generate the Fibonacci number series. In the end, the returned array always comprises 's' Fibonacci numbers.
```

Comment:
So at iteration `s`, the array would contain `s` Fibonacci numbers? That doesn't looks like it's true.